app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -174,7 +176,19 @@ def send_failed_email(user, enrollment)
         user,
         email_address,
         enrollment: enrollment,
-      ).deliver_now_or_later
+      ).deliver_now_or_later(**mail_delivery_params)
+    end
+  end
+
+  def mail_delivery_params
+    config_delay = IdentityConfig.store.in_person_results_delay_in_hours
+    if config_delay.present?
```

Comment:
When would the config value not be present?

Reply:
oh yeah, I guess that's impossible, because the values are set to the defaults in `config/application.yml.default`; I'll remove that condition.

Reply:
done in a4f71ff4a63a9591e6c3218839cca33e410d58a8